src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -87,17 +88,13 @@ public String addSessionTokenToUrl(String link) {
      * Returns the timezone options as HTML code.
      * None is selected, since the selection should only be done in client side.
      */
-    protected List<String> getTimeZoneOptionsAsHtml(double existingTimeZone) {
-        List<Double> options = TimeHelper.getTimeZoneValues();
+    protected List<String> getTimeZoneOptionsAsHtml(ZoneId existingTimeZone) {
+        List<ZoneId> options = TimeHelper.getTimeZoneValues();
         ArrayList<String> result = new ArrayList<>();
-        if (existingTimeZone == Const.DOUBLE_UNINITIALIZED) {
-            result.add("<option value=\"" + Const.INT_UNINITIALIZED + "\" selected></option>");
-        }
-        for (Double timeZoneOption : options) {
-            String utcFormatOption = StringHelper.toUtcFormat(timeZoneOption);
-            result.add("<option value=\"" + formatAsString(timeZoneOption) + "\""
-                       + (existingTimeZone == timeZoneOption ? " selected" : "") + ">" + "(" + utcFormatOption
-                       + ") " + TimeHelper.getCitiesForTimeZone(Double.toString(timeZoneOption)) + "</option>");
+        for (ZoneId timeZoneOption : options) {
+            result.add("<option value=\"" + timeZoneOption.toString() + "\""
+                    + (existingTimeZone.equals(timeZoneOption) ? " selected" : "") + ">" + "(" + timeZoneOption.toString()
```

Comment:
Any reason for using `toString` instead of `getId` these two lines? I prefer `getId` as it's more explicit.

Reply:
If you're using `toString`, might as well omit the function call as it's implied.

Reply:
Up to you whether you want to change it; this is really minor.

Reply:
Explicit is better, changed it to `getId()`